rsconf/component/raydata_scan_monitor.py

```diff
@@ -29,6 +31,7 @@ def internal_build_compile(self):
         )
         z.db_d = z._run_d.join(_DB_SUBDIR)
         z.intake_d = z._run_d.join(_INTAKE_D)
+        z.notebook_d = z._run_d.join(_NOTEBOOK_D)
```

Comment:
need to add this to jc.sirepo so it is populated in the env file.

Reply:
950e53473a38140aab077fbfeab13fe54c366e2c